module3/src/com/epam/regularexpressions/XmlParser.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.regularexpressions;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class XmlParser {
+    public static String xmlAnalyzer(String xmlText) {
```

Comment:
okay, logic is fine but please fix the output:
![image](https://user-images.githubusercontent.com/33148718/113574915-bd5ee000-9625-11eb-8252-db70bb1cc632.png)

Your values have the opening tag